src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    protected Color color;
+
+    abstract void draw();
```

Comment:
let's remove this method - you should also create `interface Drawable` with method `void draw()` and implement this interface in Figure